### ISSUE TITLE
Artifacts in URxvt with colors enabled

### ISSUE THREAD

**Issue by Barbaross93** [octobanana/octavia#2]
Running octavia in URxvt produces artifacts when enabling colors. While running octavia in tmux, everything seems to work great. Even when I try disabling alpha compositing and color shift in URxvt the problem is still there, although not as bad when both are enabled. I'm betting this is because URxvt doesn't support true color. Is there a possibility of working with Xresources color palettes in the future?

**Comment by octobanana** [comment]
Exactly, the artifacts are due to `rxvt-unicode` lacking support for true colour. If you want to use `rxvt-unicode` with true colour support, there's a patched version available in the __AUR__ called `rxvt-unicode-truecolor`. I just tried it out and the colours seem to display fine.

I'm planning on adding support for 4-bit (16 colours / Xresources palette), 8-bit (256 colours), and 24-bit (true colour) colours with different gradients and patterns.

**Comment by Barbaross93** [comment]
Excellent; I didn't realize there was a truecolor version of urxvt on the AUR, that worked for me! One last question, and maybe I missed this somewhere, but is there an option to change the colors? It would be nice to set them to my colorscheme

**Comment by octobanana** [comment]
An option to change the default colours without recompiling will come with the configuration file support, it's what I'm working on now! Until then, first make sure to update to the new [1.2 release](https://github.com/octobanana/octavia/releases/tag/0.1.2), as there's lots of improvements to the colour gradients. Then you can change the default colours in the [./src/app/app.hh](https://github.com/octobanana/octavia/blob/1c45addc4f8341bb7a8b445773efd3c8d2d10e90/src/app/app.hh#L250) file. It should be pretty straightforward to change the hex colour, if you have any questions on changing it let me know!

Also, try playing around with the `n`, `N`, `m`, and `M` keys, it's now possible to toggle the x-axis gradient and y-axis gradient individually, which can make some neat looking patterns!

**Comment by stephaneThannio** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
tiendasoficiales, @haixoay96, @marooo000, @ausov, @zxh0, @jonblanchette, @modot, @liskyli, @iemejia, @mul1sh, @Inversion-des, @torenotehus, @brunoerg, @fenix011, @Linda-chan, @rokinsky, @noooobady, @13werwolf13, @totothink, @ashokpant, @Bella70, @zzApotheosis, @hmyit, @Shubhit96, @Coldblood1004, @globalmehedi, @abstractionthief, @mfcodeworks, @wamatt, @88woods88, @zlhaa23, @BotanAtomic, @fiddler, @pprobst, @mgiraldeau, @saltire, @discosultan, @youmoula, @heyalexchoi, @kaktumar, @rfreis, @WuShichao, @ITx78, @julien-boudry, @Kayla446, @GoMino, @Necmeddinhammami, @ozefe, @Lastrellik, @ragymorkosgiulioape, @0xjogir, @ruan4261, @azizrbii90, @bautistasb, @UnidentifiedFox, @ashleyniemerg, @cercatrova21, @zhangning737, @c4k3man, @kissthink, @Armour, @bpg0x03, @zanusaurelius, @thematrixdev, @chenzw95, @RashadAnsari, @MichaelPolla, @iNemesis, @fanyer, @vinhmako99777, @dabuziduo, @kokumaji, @ommz, @slowmistio, @MohamedKiouaz, @abaez, @shahinism, @gandaro, @pygeek, @Nesquik, @zhengsen, @mihirvagal, @FastTrac71, @Th4rm, @Sambhavj023, @dylangilbert, @etil2jz, @cd365, @kikilaw, @hipro, @alexpaths, @lxl0928, @igordelorenzi, @5af64758, @lgldsilva, @ablozhou, @L0laL33tz, @austinmajor, @gerryeng
